charts/gardener-dashboard/charts/application/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+# SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+description: A Helm chart to deploy the Gardener dashboard application related components
+name: application
+version: 0.2.0
```

Comment:
We should change the version to `0.1.0` or `0.3.0`

Reply:
Changed to `0.1.0` for all charts as it does not affect the helm chart installation